src/backend/InvenTree/plugin/builtin/labels/label_sheet.py

```diff
@@ -82,6 +82,8 @@ def in_debug_mode(self):
         if self.debug is None:
             self.debug = str2bool(self.get_setting('DEBUG'))
 
+        return self.debug
```

Comment:
What is the reasoning behind first checking if self.debug is None? Surely would be simpler to just return the setting check

Reply:
I have to admit that I have actually I have copy pasted this from here:
https://github.com/inventree/InvenTree/blob/master/src/backend/InvenTree/plugin/builtin/labels/inventree_label.py#L46
I think the original author's intention was to cache the settings.

Reply:
Ok, this caching behaviour does not work as expected. Once you first print a label, if you then change the "debug" setting, the original value of the setting is still used (until the server is restarted).

It is also a bug in the implementation you copied from.

Can you please remove the caching behaviour? I have tested it locally and it is not correct.

If possible it would be great if you could also remove the caching behaviour from the other plugin where you copied it from.

Reply:
I removed caching, tested it works on the fly.
I am wondering what was the original motivation of this the caching.

 PR coming soon for the other plugin.